vars/swarm.groovy

```diff
@@ -0,0 +1,86 @@
+import groovy.json.JsonSlurper
+
+class SwarmConfig {
+    static String swarmUser = '';
+    static String swarmUrl = '';
+    static String p4Ticket = '';
+}
+
+def init(String swarmUser, String swarmUrl, String p4Ticket) {
+    SwarmConfig.swarmUser = swarmUser;
+    SwarmConfig.swarmUrl = swarmUrl;
+    SwarmConfig.p4Ticket = p4Ticket;
+}
+
+def clear() {
+    SwarmConfig.swarmUser = '';
+    SwarmConfig.swarmUrl = '';
+    SwarmConfig.p4Ticket = '';
+}
+
+def getGroupMembers(groups, String groupName) {
+    return groups.getGroupMembers(groups, groupName).keySet() as String[];
+}
+
+def getGroupMembers(groups, List<String> groupNames) {
```

Comment:
Two functions defined as "getGroupMembers"?

Reply:
I decided to overload it so that users don't have to use `[groupName]` if they only want to look up members from one group.